run-tests.sh

```diff
@@ -54,7 +54,7 @@ fi
 
 python -m check_manifest
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
-eval "$(docker-services-cli up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --mq ${MQ:-redis} --env)"
+eval "$(docker services up --db ${DB:-postgresql} --search ${SEARCH:-opensearch} --mq ${MQ:-redis} --env)"
```

Comment:
The script run-tests.sh has a command that I don’t think is available on OSX. I tried making some changes to see if I could find a match for OSX, but that’s going nowhere. I think the move might be to spin up containers specifically for this, but I’m not sure where to find the files that define the containers for these tests. The --env flag seems to imply there is a Dockerfile or some other type of file that is supposed to be around. I was thinking it might make sense to just write my own and fill it in with the container definitions I need, but that seems wrong.

Reply:
Installing test dependencies should provide you with the `docker-services-cli` CLI utility: `pipenv run pip install -e .[tests,opensearch2]` . This is an invenio utility, not a docker provided thing and should work on MacOS. So these sections shouldn't be modified.

Reply:
I'll give that another shot. I was pretty sure I had run setup tools correctly, but there's a good chance I did something wrong and then went this painful road of ignorance and confusion.